.github/workflows/integrationTests.yaml

```diff
@@ -48,12 +48,15 @@ jobs:
         with:
           java-version: 11
       - name: Setup GraalVM
-        uses: rinx/setup-graalvm-ce@v0.0.5
+        uses: DeLaGuardo/setup-graalvm@4.0
         if: matrix.it == 'native'
         with:
-          graalvm-version: "21.2.0"
-          java-version: "java11"
-          native-image: "true"
+          graalvm: '21.2.0'
+          java: 'java11'
+          arch: 'amd64'
+      - name: Install GraalVM Native Image
+        if: matrix.it == 'native'
+        run: gu install native-image
       - uses: actions/cache@v2
         id: mvn-cache
```

Comment:
My 2 cents: if we cache the GraalVM native image installation too, this job may become a bit faster. (I don't know the bottleneck)